cpp/test/compat/core/compat.cmake

```diff
@@ -20,3 +20,20 @@ if (CSHARP_COMPAT)
         run (${CSHARP_COMPAT} ${TEST} compat.Compat.json compat.${TEST}.gbc.dat)
     endif()
 endif()
+
+if (JAVA_COMPAT)
+    if (WIN32)
+        set (PATHSEP "\\\;")
+    else()
+        set (PATHSEP ":")
+    endif()
+
+    run (java -classpath ${JAVA_CORE}${PATHSEP}${JAVA_COMPAT} com.microsoft.bond.compat.CompatDriver
```

Comment:
I just tried doing a build on a Linux desktop on a folder with a space in it... we have a lot of these problems.

I was building out of a directory called ```bond space```, my build failed very early because cpps weren't getting copied to the build directory, and I ended up with a directory called ```bond\\ space``` parallel to my root. That's where they went.

As-is, this is probably going to fail, but it's deficient in exactly the way the C# equivalent is, and we may need to change a lot more to even get to those failures. I'd like to leave it.